DataConnectors/OSSEC/Connector_CEF_OSSEC.json

```diff
@@ -0,0 +1,115 @@
+{
+    "id": "OSSEC",
+    "title": "OSSEC",
+    "publisher": "OSSEC",
+    "descriptionMarkdown": "OSSEC data connector provides the capability to ingest [OSSEC](https://www.ossec.net/) events into Azure Sentinel. Refer to [OSSEC documentation](https://www.ossec.net/docs) for more information.",
+    "additionalRequirementBanner": "This data connector depends on a parser based on Kusto Function to work as expected. Follow the steps to use this Kusto Function alias **OSSECEvent** in queries and workbooks. [Follow steps to get this Kusto Function>](https://aka.ms/sentinel-OSSEC-parser)  ",
+    "graphQueries": [
+        {
+            "metricName": "Total data received",
+            "legend": "OSSEC Events",
+            "baseQuery": "OSSECEvent"
+        }
+    ],
+    "sampleQueries": [
+        {
+            "description" : "Top 10 Rules", 
+            "query": "OSSECEvent\n | summarize count() by RuleName\n | top 10 by count_"
+        }
+    ],
+    "dataTypes": [
+        {
+            "name": "CommonSecurityLog (OSSEC)",
+            "lastDataReceivedQuery": "OSSECEvent\n | summarize Time = max(TimeGenerated)\n| where isnotempty(Time)"
+        }
+    ],
+    "connectivityCriterias": [
+        {
+            "type": "IsConnectedQuery",
+            "value": [
+                "OSSECEvent\n | summarize LastLogReceived = max(TimeGenerated)\n| project IsConnected = LastLogReceived > ago(30d)"
+            ]
+        }
+    ],
+    "availability": {
+        "status": 2,
+        "isPreview": true
+    },
+    "permissions": {
+        "resourceProvider": [
+            {
+                "provider": "Microsoft.OperationalInsights/workspaces",
+                "permissionsDisplayText": "read and write permissions are required.",
+                "providerDisplayName": "Workspace",
+                "scope": "Workspace",
+                "requiredPermissions": {
+                    "read": true,
+                    "write": true,
+                    "delete": true
+                }
+            },
+            {
+                "provider": "Microsoft.OperationalInsights/workspaces/sharedKeys",
+                "permissionsDisplayText": "read permissions to shared keys for the workspace are required. [See the documentation to learn more about workspace keys](https://docs.microsoft.com/azure/azure-monitor/platform/agent-windows#obtain-workspace-id-and-key).",
+                "providerDisplayName": "Keys",
+                "scope": "Workspace",
+                "requiredPermissions": {
+                    "action": true
+                }
+            }
+        ]
+    },
+    "instructionSteps": [
+        {
+            "title": "1. Linux Syslog agent configuration",

```

Comment:
Since this connector is dependent on the parser, you need to add additional instruction about the Kusto function. 
Please refer https://github.com/Azure/Azure-Sentinel/blob/master/DataConnectors/Templates/Connector_Syslog_template.json, get the first instruction from here with description ">This data connector depends on a parser based on a Kusto Function to work as expected.......".

Reply:
updated